src/averageAges.js

```diff
@@ -15,11 +23,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter(({ sex }) => sex === SEX_MALE);
+
+  if (century) {
+    men = men.filter(({ died }) =>
+      Math.ceil(died / CENTURY_DIVIDER) === century);
+  }
+
+
+  return summOfAge(men) / men.length;
```

Comment:
this part `/ men.length` also can be moved inside function, it also the same in both functions